### ISSUE TITLE
the loss doesn't decrease

### ISSUE THREAD

**Issue by burcehan** [jik876/hifi-gan#17]
Hi ,Thanks for your work
I have trained １0k steps start from scratch,the loss of generator is 8.6, the loss of discriminatoris 4.5,the learning rate is 1e-5, I found the loss doesn't decrease,I don't know why．
Thanks

**Comment by burcehan** [comment]
I use the config of v1

**Comment by CookiePPP** [comment]
Use `mel_spec_error` to examine progress, the generator and discriminator losses are not meant to decrease.

**Comment by jik876** [comment]
I close this as there are no recent updates. Please reopen if you need additional comments.
